uktrade/data-hub-frontend
# Create edit company exports form

## Description of change

Part of a task to rebuild the exports edit form with React, this is PR 1 of 2. This adds a new edit link to the "Exports" table and when clicking it there is a separate form to edit just the "Exports" data - which is currently just the "Export win category"

## Test instructions

View the export tab for a company and click edit next to "Exports". Also click the "Edit export countries" button from the table below and confirm the "Export win category" has been removed from this page and the editing of countries still works as expected

## Screenshots
### Before

<img width="838" alt="Screenshot 2020-03-16 at 14 26 35" src="https://user-images.githubusercontent.com/1481883/76767960-4a709600-6792-11ea-9889-f88552e3cf05.png">

<img width="789" alt="Screenshot 2020-03-16 at 14 26 50" src="https://user-images.githubusercontent.com/1481883/76767981-4e041d00-6792-11ea-8b6c-71e381bb785d.png">


### After

<img width="832" alt="Screenshot 2020-03-16 at 14 24 25" src="https://user-images.githubusercontent.com/1481883/76768005-53f9fe00-6792-11ea-83f0-bc4a3aa6469c.png">

<img width="433" alt="Screenshot 2020-03-16 at 14 24 42" src="https://user-images.githubusercontent.com/1481883/76768014-578d8500-6792-11ea-8099-ad0ff6dc4f79.png">

<img width="782" alt="Screenshot 2020-03-16 at 14 24 54" src="https://user-images.githubusercontent.com/1481883/76768024-5b210c00-6792-11ea-958c-10ec4b740e96.png">



## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
